client/providers/CallProvider/CallProvider.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useMemo, FC, useRef } from 'react';
+import { createPortal } from 'react-dom';
+
+import { CallContext, CallContextValue } from '../../contexts/CallContext';
+import {
+	isUseVoipClientResultError,
+	isUseVoipClientResultLoading,
+	useVoipClient,
+} from './hooks/useVoipClient';
+
+export const CallProvider: FC = ({ children }) => {
+	// TODO: Test Settings and return false if its disabled (based on the settings)
```

Comment:
is not tested yet?

Reply:
We will have to do some changes to the way we are handling configuration and settings, so not yet